packages/frontend/components/verification/enter-phone/component.tsx

```diff
@@ -29,7 +32,14 @@ const EnterPhoneModal = ({ phone, setPhone }: ModalProps) => {
   };
 
   const handleOnChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setPhone(e.target.value);
+    setphoneInput(e.target.value);
+    if (e.target.value != phone) {
+      setDisabled(true);
+      setTooltipText(t('common:verify.enterPhone.tooltip'));
+    } else {
+      setDisabled(false);
```

Comment:
maybe you can use tooltipText like a trigger for disabling button => disabled={!!tooltipText}